assets/js/analytics-order.js

```diff
@@ -1,42 +1,38 @@
 (() => {
     "use strict";
-    const e = wp.hooks,
-        r = wp.i18n;
-    function t(e) {
+    function r(r) {
```

Comment:
Shouldn't the function names be more explicit? I haven't checked WC code, is this something they do as well?

Reply:
I created this version using `pnpm` and unminified it for the GitHub action.
The file was built from the `analytics-order.mjs` script, which should be edited directly (I renamed it to .mjs to avoid github action error because of the `import` usage in the script).
So I think keeping the file as it is maybe appropriate, but editing would also be ok.
The WC uses the build version, such as `assets/client/admin/chunks/analytics-report-orders.js` for orders in analytics.
Also, it is better not to include the `.mjs` file from production since it is unnecessary and only for developers' future reference.
What do you think?

Reply:
Ok, let's keep like that.